packages/tools/pluggable-widgets-tools/scripts/e2e.js

```diff
@@ -63,17 +67,27 @@ async function main() {
         `docker run -td -v ${process.cwd()}:/source -v ${__dirname}:/shared:ro -w /source -p ${freePort}:8080 ` +
             `-u root -e MENDIX_VERSION=${latestMendixVersion} --entrypoint /bin/bash ` +
             `mendix/runtime-base:${latestMendixVersion}-rhel /shared/runtime.sh`
-    ).toString();
+    )
+        .toString()
+        .trim();
+
+    // Spin up the standalone selenium firefox
+    const freePortFirefox = await findFreePort(4444);
```

Comment:
That means that locally you will also run firefox in container. Are we sure we want that?

Reply:
Absolutely sure, it was failing on screenshot tests because the environments were different. Even in travis